odxtools/diaglayers/diaglayerraw.py

```diff
@@ -77,10 +77,6 @@ def from_et(et_element: ElementTree.Element, context: OdxDocContext) -> "DiagLay
             variant_type = cast(DiagLayerType, None)
             odxraise(f"Encountered unknown diagnostic layer type '{et_element.tag}'")
 
-        short_name = odxrequire(et_element.findtext("SHORT-NAME"))
-
-        # extend the applicable ODX "document fragments" for the diag layer objects
-        context.doc_fragments.append(OdxDocFragment(short_name, DocType.LAYER))
```

Comment:
what happened to this logic ?

Reply:
The LAYER doc fragment is now created in DiagLayerContainer.